stdlib/date/date.flux

```diff
@@ -39,6 +39,35 @@ package date
 // ```
 builtin second : (t: T) => int where T: Timeable
 
+// time returns the datetime of a specified timeable.
+//
+// ## Parameters
+// - t: Timeable. It can be either time or duration
+//
+//   Use an absolute time or relative duration.
+//   Durations are relative to `now()`.
+//
+// ## Examples
+//
+// ### Return the time for a given time
+//
+// ```no_run
+// import "date"
+//
+// date.time(t: 2020-02-11T12:21:03.293534940Z)
+// // Returns 2020-02-11T12:21:03.293534940Z
+// ```
+// ### Return the time for a given relative duration
+//
+// ```no_run
+// import "date"
+//
+// date.time(t: -1d1h1m1s1us1ns)
+//
+// // Returns 2022-06-02T20:02:19.506570999Z
+// ```
```

Comment:
- To keep this example and return value constant, define the `now` option in the example.
- Add the `introduced: NEXT` metadata to tag this function with the next tagged Flux version.

~~~suggestion
// ```no_run
// import "date"
//
// option now = () => 2022-01-01T00:00:00Z
//
// date.time(t: -1h)
//
// // Returns 2021-12-31T23:00:00.000000000Z
// ```
//
// ## Metadata
// introduced: NEXT
//
~~~